### PR TITLE
[Doc] Add multi-image input example and update supported models

### PR DESCRIPTION
**Pull request by DarkLight1337**
This PR updates the VLM documentation with examples for multi-image input.

It also updates the list of supported multi-modal models to indicate whether embeddings and multi-input are supported for each modality.

### CODE REVIEW

File: docs/source/models/vlm.rst [hunk 1]
```diff
@@ -9,10 +9,6 @@ This document shows you how to run and serve these models using vLLM.
 .. important::
     We are actively iterating on VLM support. Expect breaking changes to VLM usage and development in upcoming releases without prior deprecation.
 
-    Currently, the support for vision language models on vLLM has the following limitations:
-
-    * Only single image input is supported per text prompt.
```
Comment (ywang96) [inline docs/source/models/vlm.rst:14]: Perhaps we can add a line to say users can refer to the supported model section to see which models actually work with multiple images input.

**Review by ywang96 — APPROVED** [review]
LGTM! Left a comment

File: docs/source/models/vlm.rst (hunk 1 above)
Comment (DarkLight1337) [reply docs/source/models/vlm.rst:14]: I'm thinking of updating the docs to include an example of multi-image input, but unfortunately LLaVA-1.5 isn't really a good candidate for showcasing this. What do you think of switching to Phi-3.5-Vision for the example?

Comment (ywang96) [reply docs/source/models/vlm.rst:14]: Yea I actually updated the vision api client example with phi3.5v instruct, so feel free to just use that one.

Comment (DarkLight1337) [reply docs/source/models/vlm.rst:14]: Ok let me try to do it in this PR then since it already exists.

Comment (DarkLight1337) [reply docs/source/models/vlm.rst:14]: Done, please take a look.

Comment (ywang96) [reply docs/source/models/vlm.rst:14]: Overall looks good to me. Feel free to merge whenever you find it ready!

**Pull request merged**